Apps/AdminWebClient/src/Server/Models/CovidSupport/CovidAssessmentRequest.cs

```diff
@@ -88,8 +88,8 @@ public CovidAssessmentRequest()
     /// <summary>
     /// Gets or sets the identifies the confirms over 12 option.
     /// </summary>
-    [JsonPropertyName("confirmsOver12")]
-    public CovidTherapyAssessmentOption ConfirmsOver12 { get; set; }
+    [JsonPropertyName("confirmsOver18")]
+    public CovidTherapyAssessmentOption ConfirmsOver18 { get; set; }
```

Comment:
The documentation for this property should be updated to reference the change in age as well.

We should also confirm with @ninosamson that PHSA is updating the name of their property, otherwise this will break.